package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mx-design-tokens",
-  "version": "10.1.1",
+  "version": "10.1.2",
```

Comment:
Since these are new tokens I think this should be `10.2.0`. You will need to run the `npm run build:all` after you update the version because that gets added to the `.scss` files automatically.